VideoLocker/res/values/styles.xml

```diff
@@ -252,6 +256,15 @@
         <item name="android:textSize">@dimen/edx_small</item>
     </style>
 
+    <style name="edX.Widget.BorderButton">
```

Comment:
This should probable be named `BorderedButton` instead (and also the background drawable), to be more semantically correct.